### PR TITLE
PullRequest_arrow

### PR DESCRIPTION
**Pull request by santosh653**
Creating arrow.travis.yml with proposed changes.

### CODE REVIEW

**Comment by seth-priya** [comment]
@santosh653 I don't think what we have here is the full set of builds though I could not confirm as the community CI is off. I think we should probably be emulating s390x in this case, @Siddhesh-Ghadi your thoughts?

File: travis-ymls/arrow.travis.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+dist: bionic
+
+language: minimal
+arch:
+ - amd64
+ - ppc64le
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/arrow.travis.yml:29]: Remove this 
```suggestion
```
and add jobs similar to `s390x` under `jobs`

**Comment by gururajrkatti** [comment]
@santosh653 any updates here ?

**Comment by gururajrkatti** [comment]
@santosh653 any updates on this ?

**Comment by santosh653** [comment]
closing this PR as there were failures on power.,

**Pull request closed without merging**